cirq-core/cirq/ops/phased_iswap_gate_test.py

```diff
@@ -25,19 +25,26 @@
 def test_phased_iswap_init():
     p = -0.25
     t = 0.75
-    gate = cirq.PhasedISwapPowGate(phase_exponent=p, exponent=t)
+    s = 0.5
+    gate = cirq.PhasedISwapPowGate(phase_exponent=p, exponent=t, global_shift=s)
     assert gate.phase_exponent == p
     assert gate.exponent == t
+    assert gate.global_shift == s
 
 
 def test_phased_iswap_equality():
     assert cirq.PhasedISwapPowGate(phase_exponent=0, exponent=0.4) == cirq.ISWAP**0.4
+    assert (
```

Comment:
> Add tests for value equality of gates with non-zero global shift (which should work correctly because it depends upon `self._iswap._value_equality_values_`)

Done.

Reply:
Also add a test for inequality
```
cirq.PhasedISwapPowGate(phase_exponent=0, exponent=0.4, global_shift=0.3) != cirq.ISWAP ** 0.4
```

It would be better to use EqualsTester if we can.

Reply:
> It would be better to use EqualsTester if we can.

Done.